_data/groups/CodeandStuff.yml

```diff
@@ -0,0 +1,13 @@
+description: |
+    A coding and networking community for women and non-binary developers and code newbies in Manchester.
+    The group meets weekly, and is free to attend, but
+    require tickets via Eventbrite.
+organiser: Fey Ijaware
+email: codeandstuff.manc"gmail.com
```

Comment:
Suspect this should have been codeandstuff.manc@gmail.com ?

Reply:
yes thanks